src/main/java/seedu/weme/ui/MemeCard.java

```diff
@@ -85,6 +108,40 @@ public void update(Meme meme, int newIndex, SimpleIntegerProperty numOfLikes, Si
                 dislikes.setText(Integer.toString((int) newValue)));
     }
 
+    private int getTagLimit(Meme meme) {
```

Comment:
Explain what this method does pls, or at least the reasoning behind it

Reply:
this method gets the max number of tags to be displayed s.t. the content won't overflow.